pgcli/pgcompleter.py

```diff
@@ -61,18 +62,38 @@ def Candidate(
 normalize_ref = lambda ref: ref if ref[0] == '"' else '"' + ref.lower() + '"'
 
 
-def generate_alias(tbl):
+def generate_alias(tbl, alias_map=None):
```

Comment:
Looks like we have no tests for this functions. I'd add one, especially now that the behavior is getting more complicated.